### ISSUE TITLE
Focus is lost on clicking message bar close button

### ISSUE THREAD

**Issue by mdmdakbar** [OfficeDev/office-ui-fabric-react#3329]
Priority: Medium

Products/sites affected:  VSTS

Actual behavior:
On clicking the message bar close button, the focus is lost.

Expected behavior:
On clicking the message bar close button, focus should be on previously focused element in the page.

Please let me know if you need any more data on this issue. This issue is generic and happening in all places where message bar component is used.

**Comment by Jahnp** [comment]
Hey @mdmdakbar , managing focus in this context sounds like an app-level concern, not something that'd be handled by Fabric. You should be able to re-assign focus where you need to with MessageBar's `onDismiss`. See "implementation" on [MessageBar's docs site page](https://developer.microsoft.com/en-us/fabric#/components/messagebar). I'm going to go ahead and close this for now, but let me know if you feel there's a real issue with Fabric here.
